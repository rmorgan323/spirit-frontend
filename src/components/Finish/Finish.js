import React, { Component } from 'react';
import './Finish.css';
import TreatmentPlans from '../TreatmentPlans/TreatmentPlans';
import TherapyGoals from '../../containers/TherapyGoals/TherapyGoals';
import { connect } from 'react-redux';
import * as actions from '../../actions';

class Finish extends Component {

  completeSessionNow = () => {
    this.props.updateSession(this.props.selectedSession.id, {completed: true})
  }

  render() {
    return (
      <div className="Finish">
        <TreatmentPlans />

        <TherapyGoals category="modulation_" title="Modulation" number="1" />

        <TherapyGoals category="posture_" title="Postural" number="2" />

        <TherapyGoals
          category="sensory_discrimination_"
          title="Sensory Discrimination"
          number="3"
        />

        <TherapyGoals
          category="social_emotional_"
          title="Social/Emotional Development"
          number="4"
        />

        <TherapyGoals
          category="executive_functioning_"
          title="Executive Functioning"
          number="5"
        />

        <button 
          className="complete-session-button"
          onClick={() => this.completeSessionNow()}
        >COMPLETE SESSION</button>
      </div>
    );
  }
}

const mapStateToProps = store => ({
  selectedSession: store.selectedSession
})

const mapDispatchToProps = dispatch => ({
  updateSession: (sessionId, status) => {
    dispatch(actions.updateSession(sessionId, status))
  }
})

export default connect(mapStateToProps, mapDispatchToProps)(Finish);
