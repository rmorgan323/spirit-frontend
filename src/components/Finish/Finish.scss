@import '../../variables.scss';

.Finish {
  .complete-session-button {
    height: 40px;
    width: 199px;
    background-color: $purple4;
    border: none;
    color: #fff;
    margin: 50px auto 100px;
    display: block;
    cursor: pointer;
    @include std-transition;
  }

  .complete-session-button:hover {
    background-color: $purple6;
  }
}