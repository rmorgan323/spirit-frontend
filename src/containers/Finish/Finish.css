/******  GLOBAL CSS VARIABLES  ******/
/***  COLORS  ***/
/***  FONTS  ***/
* {
  font-size: 16px;
  font-family: "Lato", sans-serif;
  outline: none;
  color: #6d6e71; }

h1, h2, h3, h4, h5, h6 {
  font-family: "Josefin Sans", sans-serif;
  line-height: 1.3; }

h1 {
  font-size: 44px; }

h2 {
  font-size: 30px; }

h3 {
  font-size: 24px; }

h4 {
  font-size: 18px; }

h5 {
  font-size: 14px; }

h6 {
  font-size: 10px; }

p {
  font-size: 14px;
  line-height: 1.3;
  font-weight: 900; }

::placeholder {
  opacity: .5;
  text-align: center; }

/***  MIXINS  ***/
.Finish .complete-session-button {
  height: 40px;
  width: 199px;
  background-color: #9E87BA;
  border: none;
  color: #fff;
  margin: 50px auto 100px;
  display: block;
  cursor: pointer;
  transition: all .2s ease-in-out; }

.Finish .complete-session-button:hover {
  background-color: #5E378C; }

.Finish .finish-headings, .Finish .finish-sub-headings {
  text-align: center; }

.Finish .dividing-line {
  width: 675px;
  height: 2px;
  margin: auto;
  background: #6d6e71; }

.Finish .finish-button-holder {
  position: relative;
  width: 875px;
  margin: auto;
  height: 40px;
  display: flex;
  align-items: center;
  justify-content: flex-end;
  margin-bottom: 20px; }
  .Finish .finish-button-holder .arrow-right {
    margin-right: 10px;
    margin-left: 10px; }

.Finish .join-link button {
  background-color: #5cba47;
  height: 40px;
  border: none;
  color: #fff;
  padding: 0 10px;
  cursor: pointer; }

.Finish .join-link button:hover {
  background-color: #468c41; }
