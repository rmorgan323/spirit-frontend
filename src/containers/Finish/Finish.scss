@import '../../variables.scss';

.Finish {
  .complete-session-button {
    height: 40px;
    width: 199px;
    background-color: $purple4;
    border: none;
    color: #fff;
    margin: 50px auto 100px;
    display: block;
    cursor: pointer;
    @include std-transition;
  }

  .complete-session-button:hover {
    background-color: $purple6;
  }

  .finish-headings, .finish-sub-headings {
    text-align: center;
  }

  .dividing-line {
    width: 675px;
    height: 2px;
    margin: auto;
    background: $grey5;
  }

  .finish-button-holder {
    position: relative;
    width: 875px;
    margin: auto;
    height: 40px;
    display: flex;
    align-items: center;
    justify-content: flex-end;
    margin-bottom: 20px;

    .arrow-right {
      margin-right: 10px;
      margin-left: 10px;
    }
  }

  .join-link {
    button {
      background-color: $green4;
      height: 40px;
      border: none;
      color: #fff;
      padding: 0 10px;
      cursor: pointer;
    }

    button:hover {
      background-color: $green5;
    }
  }
}