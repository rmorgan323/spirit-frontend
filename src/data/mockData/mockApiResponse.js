const mockApiResponse = {
  status: 'success'
};

export default mockApiResponse;