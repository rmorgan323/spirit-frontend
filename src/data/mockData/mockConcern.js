/*eslint-disable camelcase*/

export const mockConcern = {
  id: 5,
  description: "Concern 1",
  domain_1: true,
  domain_2: true,
  domain_3: false,
  domain_4: true,
  domain_5: false,
  domain_6: false,
  notes: "Some notes.",
  patient_id: 5,
  created_at: "2018-02-20T20:08:19.142Z",
  updated_at: "2018-02-20T20:08:19.142Z"
};
