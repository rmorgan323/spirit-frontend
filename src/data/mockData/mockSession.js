/*eslint-disable camelcase*/

export const mockSession = {
  id: 10,
  concern_id: 5,
  created_at: "2018-02-20T21:25:45.411Z",
  updated_at: "2018-02-20T21:25:45.411Z",
  completed: true
};
