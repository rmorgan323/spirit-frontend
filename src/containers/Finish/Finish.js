import React, { Component } from 'react';
import { connect } from 'react-redux';
import { NavLink } from 'react-router-dom';
import { PropTypes } from 'prop-types';
import TreatmentPlans from '../../components/TreatmentPlans/TreatmentPlans';
import TherapyGoals from '../TherapyGoals/TherapyGoals';
import { finish } from '../../language/language';
import './Finish.css';

export class Finish extends Component {
  render() {
    return (
      <div className="Finish">
        <div className="finish-button-holder">
          <p>
            {finish.recommend}
            <span className="arrow-right">&#10145;</span>
          </p>
          <NavLink
            className="join-link"
            to={`/spirit/sessions/${this.props.selectedSession.id}/view`}
          >
            <button className="session-result-button">
              SEE SESSION RESULTS
            </button>
          </NavLink>
        </div>

        <h2 className="finish-headings">Treatment Plans</h2>
        <div className="dividing-line" />
        <h5 className="finish-sub-headings">{finish.stepsi}</h5>

        <TreatmentPlans />

        <h2 className="finish-headings">Therapy Goals</h2>
        <div className="dividing-line" />
        <h5 className="finish-sub-headings">{finish.copm}</h5>

        <TherapyGoals category="modulation_" title="Modulation" number="1" />

        <TherapyGoals category="posture_" title="Postural" number="2" />

        <TherapyGoals
          category="sensory_discrimination_"
          title="Sensory Discrimination"
          number="3"
        />

        <TherapyGoals
          category="social_emotional_"
          title="Social/Emotional Development"
          number="4"
        />

        <TherapyGoals
          category="executive_functioning_"
          title="Executive Functioning"
          number="5"
        />

        <div className="finish-button-holder">
          <NavLink
            className="join-link"
            to={`/spirit/sessions/${this.props.selectedSession.id}/view`}
          >
            <button className="session-result-button">
              SEE SESSION RESULTS
            </button>
          </NavLink>
        </div>
      </div>
    );
  }
}

export const mapStateToProps = store => ({
  selectedSession: store.selectedSession,
  selectedConcern: store.selectedConcern
});

export default connect(
  mapStateToProps,
  null
)(Finish);

Finish.propTypes = {
  updateSession: PropTypes.func,
  selectedSession: PropTypes.object,
  history: PropTypes.object,
  selectedConcern: PropTypes.object
};
