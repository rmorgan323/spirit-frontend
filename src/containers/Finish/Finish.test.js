import React from 'react';
import { Finish, mapStateToProps } from './Finish';
import { shallow } from 'enzyme';
import { mockSession } from '../../data/mockData/mockSession';
import { mockConcern } from '../../data/mockData/mockConcern';

describe('Finish tests', () => {
  let selectedSession;
  let selectedConcern;
  let updateSession;
  let renderedFinish;

  beforeEach(() => {
    selectedSession = mockSession;
    selectedConcern = mockConcern;
    updateSession = jest.fn();

    renderedFinish = shallow(
      <Finish
        selectedSession={selectedSession}
        selectedConcern={selectedConcern}
        updateSession={updateSession}
      />
    );
  });

  it('Should match the snapshot', () => {
    expect(renderedFinish).toMatchSnapshot();
  });
});

describe('mapStateToProps tests', () => {
  it('Should pull selectedSession and selectedConcern from store', () => {
    const selectedSession = mockSession;
    const selectedConcern = mockConcern;
    const mockStore = { selectedSession, selectedConcern };
    const result = mapStateToProps(mockStore);

    expect(result.selectedSession).toEqual(mockStore.selectedSession);
    expect(result.selectedConcern).toEqual(mockStore.selectedConcern);
  });
});
