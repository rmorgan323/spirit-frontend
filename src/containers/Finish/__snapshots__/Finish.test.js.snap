// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Finish tests Should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Finish
    selectedConcern={
      Object {
        "created_at": "2018-02-20T20:08:19.142Z",
        "description": "Concern 1",
        "domain_1": true,
        "domain_2": true,
        "domain_3": false,
        "domain_4": true,
        "domain_5": false,
        "domain_6": false,
        "id": 5,
        "notes": "Some notes.",
        "patient_id": 5,
        "updated_at": "2018-02-20T20:08:19.142Z",
      }
    }
    selectedSession={
      Object {
        "completed": true,
        "concern_id": 5,
        "created_at": "2018-02-20T21:25:45.411Z",
        "id": 10,
        "updated_at": "2018-02-20T21:25:45.411Z",
      }
    }
    updateSession={[Function]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="finish-button-holder"
        >
          <p>
            We recommend reviewing your session results before writing plans and goals
            <span
              className="arrow-right"
            >
              ➡
            </span>
          </p>
          <NavLink
            activeClassName="active"
            aria-current="page"
            className="join-link"
            to="/spirit/sessions/10/view"
          >
            <button
              className="session-result-button"
            >
              SEE SESSION RESULTS
            </button>
          </NavLink>
        </div>,
        <h2
          className="finish-headings"
        >
          Treatment Plans
        </h2>,
        <div
          className="dividing-line"
        />,
        <h5
          className="finish-sub-headings"
        >
          S.T.E.P.S.I. ©
        </h5>,
        <TreatmentPlans />,
        <h2
          className="finish-headings"
        >
          Therapy Goals
        </h2>,
        <div
          className="dividing-line"
        />,
        <h5
          className="finish-sub-headings"
        >
          Modified from the C.O.P.M.
        </h5>,
        <Connect(TherapyGoals)
          category="modulation_"
          number="1"
          title="Modulation"
        />,
        <Connect(TherapyGoals)
          category="posture_"
          number="2"
          title="Postural"
        />,
        <Connect(TherapyGoals)
          category="sensory_discrimination_"
          number="3"
          title="Sensory Discrimination"
        />,
        <Connect(TherapyGoals)
          category="social_emotional_"
          number="4"
          title="Social/Emotional Development"
        />,
        <Connect(TherapyGoals)
          category="executive_functioning_"
          number="5"
          title="Executive Functioning"
        />,
        <div
          className="finish-button-holder"
        >
          <NavLink
            activeClassName="active"
            aria-current="page"
            className="join-link"
            to="/spirit/sessions/10/view"
          >
            <button
              className="session-result-button"
            >
              SEE SESSION RESULTS
            </button>
          </NavLink>
        </div>,
      ],
      "className": "Finish",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p>
              We recommend reviewing your session results before writing plans and goals
              <span
                className="arrow-right"
              >
                ➡
              </span>
            </p>,
            <NavLink
              activeClassName="active"
              aria-current="page"
              className="join-link"
              to="/spirit/sessions/10/view"
            >
              <button
                className="session-result-button"
              >
                SEE SESSION RESULTS
              </button>
            </NavLink>,
          ],
          "className": "finish-button-holder",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "We recommend reviewing your session results before writing plans and goals",
                <span
                  className="arrow-right"
                >
                  ➡
                </span>,
              ],
            },
            "ref": null,
            "rendered": Array [
              "We recommend reviewing your session results before writing plans and goals",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "➡",
                  "className": "arrow-right",
                },
                "ref": null,
                "rendered": "➡",
                "type": "span",
              },
            ],
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": <button
                className="session-result-button"
              >
                SEE SESSION RESULTS
              </button>,
              "className": "join-link",
              "to": "/spirit/sessions/10/view",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "SEE SESSION RESULTS",
                "className": "session-result-button",
              },
              "ref": null,
              "rendered": "SEE SESSION RESULTS",
              "type": "button",
            },
            "type": [Function],
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Treatment Plans",
          "className": "finish-headings",
        },
        "ref": null,
        "rendered": "Treatment Plans",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "dividing-line",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "S.T.E.P.S.I. ©",
          "className": "finish-sub-headings",
        },
        "ref": null,
        "rendered": "S.T.E.P.S.I. ©",
        "type": "h5",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Therapy Goals",
          "className": "finish-headings",
        },
        "ref": null,
        "rendered": "Therapy Goals",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "dividing-line",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Modified from the C.O.P.M.",
          "className": "finish-sub-headings",
        },
        "ref": null,
        "rendered": "Modified from the C.O.P.M.",
        "type": "h5",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "category": "modulation_",
          "number": "1",
          "title": "Modulation",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "category": "posture_",
          "number": "2",
          "title": "Postural",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "category": "sensory_discrimination_",
          "number": "3",
          "title": "Sensory Discrimination",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "category": "social_emotional_",
          "number": "4",
          "title": "Social/Emotional Development",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "category": "executive_functioning_",
          "number": "5",
          "title": "Executive Functioning",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <NavLink
            activeClassName="active"
            aria-current="page"
            className="join-link"
            to="/spirit/sessions/10/view"
          >
            <button
              className="session-result-button"
            >
              SEE SESSION RESULTS
            </button>
          </NavLink>,
          "className": "finish-button-holder",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": <button
              className="session-result-button"
            >
              SEE SESSION RESULTS
            </button>,
            "className": "join-link",
            "to": "/spirit/sessions/10/view",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "SEE SESSION RESULTS",
              "className": "session-result-button",
            },
            "ref": null,
            "rendered": "SEE SESSION RESULTS",
            "type": "button",
          },
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="finish-button-holder"
          >
            <p>
              We recommend reviewing your session results before writing plans and goals
              <span
                className="arrow-right"
              >
                ➡
              </span>
            </p>
            <NavLink
              activeClassName="active"
              aria-current="page"
              className="join-link"
              to="/spirit/sessions/10/view"
            >
              <button
                className="session-result-button"
              >
                SEE SESSION RESULTS
              </button>
            </NavLink>
          </div>,
          <h2
            className="finish-headings"
          >
            Treatment Plans
          </h2>,
          <div
            className="dividing-line"
          />,
          <h5
            className="finish-sub-headings"
          >
            S.T.E.P.S.I. ©
          </h5>,
          <TreatmentPlans />,
          <h2
            className="finish-headings"
          >
            Therapy Goals
          </h2>,
          <div
            className="dividing-line"
          />,
          <h5
            className="finish-sub-headings"
          >
            Modified from the C.O.P.M.
          </h5>,
          <Connect(TherapyGoals)
            category="modulation_"
            number="1"
            title="Modulation"
          />,
          <Connect(TherapyGoals)
            category="posture_"
            number="2"
            title="Postural"
          />,
          <Connect(TherapyGoals)
            category="sensory_discrimination_"
            number="3"
            title="Sensory Discrimination"
          />,
          <Connect(TherapyGoals)
            category="social_emotional_"
            number="4"
            title="Social/Emotional Development"
          />,
          <Connect(TherapyGoals)
            category="executive_functioning_"
            number="5"
            title="Executive Functioning"
          />,
          <div
            className="finish-button-holder"
          >
            <NavLink
              activeClassName="active"
              aria-current="page"
              className="join-link"
              to="/spirit/sessions/10/view"
            >
              <button
                className="session-result-button"
              >
                SEE SESSION RESULTS
              </button>
            </NavLink>
          </div>,
        ],
        "className": "Finish",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p>
                We recommend reviewing your session results before writing plans and goals
                <span
                  className="arrow-right"
                >
                  ➡
                </span>
              </p>,
              <NavLink
                activeClassName="active"
                aria-current="page"
                className="join-link"
                to="/spirit/sessions/10/view"
              >
                <button
                  className="session-result-button"
                >
                  SEE SESSION RESULTS
                </button>
              </NavLink>,
            ],
            "className": "finish-button-holder",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "We recommend reviewing your session results before writing plans and goals",
                  <span
                    className="arrow-right"
                  >
                    ➡
                  </span>,
                ],
              },
              "ref": null,
              "rendered": Array [
                "We recommend reviewing your session results before writing plans and goals",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "➡",
                    "className": "arrow-right",
                  },
                  "ref": null,
                  "rendered": "➡",
                  "type": "span",
                },
              ],
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": <button
                  className="session-result-button"
                >
                  SEE SESSION RESULTS
                </button>,
                "className": "join-link",
                "to": "/spirit/sessions/10/view",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "SEE SESSION RESULTS",
                  "className": "session-result-button",
                },
                "ref": null,
                "rendered": "SEE SESSION RESULTS",
                "type": "button",
              },
              "type": [Function],
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Treatment Plans",
            "className": "finish-headings",
          },
          "ref": null,
          "rendered": "Treatment Plans",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "dividing-line",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "S.T.E.P.S.I. ©",
            "className": "finish-sub-headings",
          },
          "ref": null,
          "rendered": "S.T.E.P.S.I. ©",
          "type": "h5",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Therapy Goals",
            "className": "finish-headings",
          },
          "ref": null,
          "rendered": "Therapy Goals",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "dividing-line",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Modified from the C.O.P.M.",
            "className": "finish-sub-headings",
          },
          "ref": null,
          "rendered": "Modified from the C.O.P.M.",
          "type": "h5",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "category": "modulation_",
            "number": "1",
            "title": "Modulation",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "category": "posture_",
            "number": "2",
            "title": "Postural",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "category": "sensory_discrimination_",
            "number": "3",
            "title": "Sensory Discrimination",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "category": "social_emotional_",
            "number": "4",
            "title": "Social/Emotional Development",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "category": "executive_functioning_",
            "number": "5",
            "title": "Executive Functioning",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <NavLink
              activeClassName="active"
              aria-current="page"
              className="join-link"
              to="/spirit/sessions/10/view"
            >
              <button
                className="session-result-button"
              >
                SEE SESSION RESULTS
              </button>
            </NavLink>,
            "className": "finish-button-holder",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": <button
                className="session-result-button"
              >
                SEE SESSION RESULTS
              </button>,
              "className": "join-link",
              "to": "/spirit/sessions/10/view",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "SEE SESSION RESULTS",
                "className": "session-result-button",
              },
              "ref": null,
              "rendered": "SEE SESSION RESULTS",
              "type": "button",
            },
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
